src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Ball ball = new Ball(colorSupplier.getRandomColor(), new Random().nextInt(100));
```

Comment:
magic number
it is better to create the Random and ColorSupplier objects as a class field with the final modifier